web/src/pages/speedtest.rs

```diff
@@ -245,60 +207,62 @@ impl Component for Speedtest {
 
             <p> { "
                 One of the most interesting features of AT2 is that it allows
-                for many more transactions per second (TPS) than the most
+                for many more transactions per second (TPS) than the two most
                 popular blockchains, such as Bitcoin or Ethereum.
-                Here you can actually test it, flooding the servers with
+                Indeed, it scales in " }
+                <code class="math">{ "log(#nodes)" }</code>
+                { " meaning that when you exponentially increase the number of
+                nodes, it only linearly reduce the TPS. The two others chains
+                scale in " }
+                <code class="math">{ "#nodes" }</code>
+                { " meaning that doubling the nodes divide the TPS by two.
```

Comment:
Same as above - is that true?

Reply:
hum, no, that's wrong. TPS is actually fixed in Bitcoin (and Ethereum IMO), having a block (with a max size) every 10min. So Bitcoin doesn't scale at all.

Yeah, so I added that after discussing with Olivier, but it falls short (he really wanted the "scales in log(n)"). I can go about it explaining that at2 only need to sample the network and not validate txs with each node, would that be clearer?

Reply:
If he wants to `log(n)`, we can put it somewhere :) But `scales in log(n)` definitely means that the higher the n, the higher the throughput.

So it's `scales in 1/log(n)`, if you want so.

I wouldn't go into too much detail here. But this is also something you can discuss with Mattheo if you show him the demonstrator.

Reply:
Going for: scales in `log(n)`; and add that the other chains have a fixed TPS.